app/models/manifest.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../search/query_dsl'
```

Comment:
I thought so too, but I think the load order is what breaks things. I.e., everything in `app` might get loaded eventually, but without those explicit `require` statements I got `undefined constant` errors when running tests.